src/main/java/frc/robot/commands/ChargeStationBalance.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.subsystems.DriveTrain;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ChargeStationBalance extends CommandBase {
+
+  //private final Rotation2d m_rampAngle = Rotation2d.fromDegrees(-10); //useful if we are going to automate going on the ramp
+  private final Rotation2d m_balancedError = Rotation2d.fromDegrees(2.5); //error for what counts as balanced
+  private final double m_balancedDegrees = 0;
```

Comment:
I don't understand why we are bothering with a "setpoint" for the balance. We all know it is 0 and we are not going to change that. I would get rid of this (but others might disagree).